packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -141,6 +142,17 @@ export default class PickerPlugin implements EditorPickerPluginInterface {
         if (!isSuggesting) {
             this.setLastKnownRange(null);
         }
+        const eventData: PickerPluginSetSuggestingData = {
+            isSuggesting,
+            pickerId: this.pickerOptions.elementIdPrefix
+        }
+        this.editor.triggerEvent(
+            {
+                eventType: PluginEventType.ContentChanged,

```

Comment:
is `ContentChanged` the right event here?

Reply:
This is something I have open questions about -- there doesn't seem to be an existing way to do plugin-to-plugin messaging other than ContentChanged.

@JiuqingSong : what was your intent with this API design? Should I be introducing a different message type here for plugin events or is ContentChanged appropriate?

Reply:
If you'd like some event should be handled in this plugin only (e.g., when picker is opened, other plugins should not handle the key inputs), you can use "willHandleEventExclusively".

PickerPlugin already did this, what it missed here is to return true for InputEvent.

Reply:
I was looking at willHandleEventExclusively, but it seemed like it wouldn't make sense here because it's possible to instantiate multiple autocomplete plugins in the same editor, isn't it?